site/docs/snapshot-tag-branch.md

```diff
@@ -0,0 +1,148 @@
+# Snapshot Tagging and Branching
+
+Iceberg snapshot tagging and branching feature offers user a Git-like experience in manging table snapshots.
```

Comment:
I am wondering here about `offers user a Git-like experience`, since we don't intend to offer a full fledged git like experience, does it make sense to add `limited Git-like experience`? In order not to confuse users I guess, WDYT?

Reply:
Yes I agree it's probably not good to say Git-like given it's not completely Git-like. But `limited` sounds a bit too strong to me, I am changing the wording to the following:

> Iceberg snapshot tagging and branching feature provides users more functionalities in managing Iceberg snapshot lifecycle, and allows users to treat table snapshots just like git branch and tag.
Users can assign tags to snapshots, create new branches, set the current branch for read and write, and configure customized retention policy for them.

Please let me know what you think.